src/AavePolygonTreasuryMigration_20230801/AavePolygonTreasuryMigration_20230801.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+
+/*
+   _      ΞΞΞΞ      _
+  /_;-.__ / _\  _.-;_\
+     `-._`'`_/'`.-'
+         `\   /`
+          |  /
+         /-.(
+         \_._\
+          \ \`;
+           > |/
+          / //
+          |//
+          \(\
+           ``
+     defijesus.eth
+*/
+
+pragma solidity ^0.8.0;
+
+import 'aave-helpers/v2-config-engine/AaveV2PayloadPolygon.sol';
+import {AaveV2Polygon, AaveV2PolygonAssets} from 'aave-address-book/AaveV2Polygon.sol';
+import {AaveV3Polygon, AaveV3PolygonAssets} from 'aave-address-book/AaveV3Polygon.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {IMigrationHelper} from 'aave-address-book/common/IMigrationHelper.sol';
+
+/**
+ * @dev Redeem aTokens from Aave v2 Polygon and deposit the underlying assets into Aave v3 Polygon. 
+ * @author defijesus.eth - TokenLogic
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x1b816c12b6f547a1982198ffd0e36412390b05828b560c9edee4e8a6903c4882
+ * - Discussion: https://governance.aave.com/t/arfc-migrate-consolidate-polygon-treasury/12248
+ */
+contract AavePolygonTreasuryMigration_20230801 is AaveV2PayloadPolygon {
```

Comment:
There is no need to use the config engine here, as payload is not updating risk params, so better to not use, to not increase size

Reply:
what do you mean? Removing the AaveV2PayloadPolygon inheritance breaks a lot of stuff

Reply:
as you are not using anything from the config engine, you dont need to use AaveV2PayloadPolygon, and also not use _postExecute method. 
A normal execute method should be enough. You can take a look at this, for example: https://github.com/bgd-labs/aave-proposals/blob/main/src/AaveV2_Eth_TreasuryManagement_20230308/AaveV2_Eth_TreasuryManagement_20230308.sol